tests/back/dataloaders/test_dashai_dataset.py

```diff
@@ -239,3 +243,77 @@ def test_save_to_disk_and_load():
     assert dashai_datasetdict["train"].outputs_columns == outputs_columns
     assert dashai_datasetdict["validation"].outputs_columns == outputs_columns
     assert dashai_datasetdict["test"].outputs_columns == outputs_columns
+
+
+@pytest.fixture(name="iris_dataset")
+def prepare_iris_dataset():
+    test_dataset_path = "tests/back/dataloaders/iris.csv"
+    dataloader_test = CSVDataLoader()
+
+    with open(test_dataset_path, "r") as file:
+        csv_binary = io.BytesIO(bytes(file.read(), encoding="utf8"))
+        file = UploadFile(csv_binary)
+
+    datasetdict = dataloader_test.load_data(
+        filepath_or_buffer=file,
+        temp_path="tests/back/converters",
+        params={"separator": ","},
+    )
+
+    inputs_columns = [
+        "SepalLengthCm",
+        "SepalWidthCm",
+        "PetalLengthCm",
+        "PetalWidthCm",
+    ]
+
+    datasetdict = to_dashai_dataset(
+        datasetdict,
+        inputs_columns,
+        outputs_columns=["Species"],
+    )
+
+    return datasetdict
+
+
+@pytest.fixture(name="iris_dataset_petal_width_dropped")
+def prepare_iris_petal_width_dropped_dataset():
+    test_dataset_path = "tests/back/dataloaders/iris_petal_width_dropped.csv"
+    dataloader_test = CSVDataLoader()
+
+    with open(test_dataset_path, "r") as file:
+        csv_binary = io.BytesIO(bytes(file.read(), encoding="utf8"))
+        file = UploadFile(csv_binary)
+
+    datasetdict = dataloader_test.load_data(
+        filepath_or_buffer=file,
+        temp_path="tests/back/dataloaders",
+        params={"separator": ","},
+    )
+
+    inputs_columns = ["SepalLengthCm", "SepalWidthCm", "PetalLengthCm"]
+
+    datasetdict = to_dashai_dataset(
+        datasetdict,
+        inputs_columns,
+        outputs_columns=["Species"],
+    )
+
+    return datasetdict
+
+
+def test_remove_columns(
+    iris_dataset: DatasetDict, iris_dataset_petal_width_dropped: DatasetDict
+):
+    assert type(iris_dataset["train"]) is DashAIDataset
+    assert type(iris_dataset_petal_width_dropped["train"]) is DashAIDataset
+    train_split: DashAIDataset = iris_dataset["train"]
+    train_dropped_split: DashAIDataset = iris_dataset["train"]
```

Comment:
These are both the same datasets. I this is the correct suggestion:
```suggestion
    train_dropped_split: DashAIDataset = iris_dataset_petal_width_dropped["train"]
```
I don't know how this test is working :rofl: